lib/perimeterx.js

```diff
@@ -2,8 +2,9 @@
 
 const pxClient = require('./utils/pxclient');
 const pxlogger = require('./utils/pxlogger');
+const PxContext = require('./utils/pxcontext');
+const PxConfig = require('./pxconfig');
 const mu = require('mu2');
-var htmlEncode = require('htmlencode').htmlEncode;
```

Comment:
htmlEncode is no longer used in the project, please remove it from package.json as well...